### DISCUSSION TITLE
How to get a list of all posts?

### DISCUSSION THREAD

**Discussion by imdevarsh** [decaporg/decap-cms#6118] [Q&A] [answered]
I asked this in https://stackoverflow.com/questions/70689203/how-to-get-a-list-of-posts-in-netlify-cms but I also thought I should ask here.

Basically I have a blog collection and there are posts inside of it. I'd like to get a file with all of the posts in there, or some other method. I'm using SvelteKit for this. Thanks!

**Comment by erezrokah** [comment, marked as answer]
Hi @webdev03, Netlify CMS saves files in your git service (e.g. GitHub) repository.

Assuming you would like to list the posts during build time, this is more of a SvelteKit question, so I recommended going over https://kit.svelte.dev/docs and asking in https://discord.com/invite/yy75DKs

**Reply by imdevarsh** [reply]
I don't know how come earlier I didn't find this page but https://vitejs.dev/guide/features.html#glob-import seems to be able to get all of the files. I'll try it out soon. Thanks!

**Reply by imdevarsh** [reply]
I found a solution for my use case here, I just need to parse it a bit I think. Thanks for the help! :smile:
```js
export async function get() {
  const items = await import.meta.glob("/static/posts/*.md");
  return {
    body: Object.keys(items)
  }
}
```
(edit: it seems like netlify doesn't like it when i use `/static/posts` so i'll try another solution
